pkg/installers/kovri-install.sh

```diff
@@ -107,7 +117,7 @@ shift "$(($OPTIND -1))"
 
 # Set default resources if needed
 if [[ -z $resources ]]; then
-  resources="pkg/client pkg/config contrib/utils/kovri-bash.sh build/${bins[0]} build/${bins[1]}"
+  resources="pkg/client pkg/config contrib/utils/kovri-bash.sh build/${bins[0]}"
```

Comment:
This isn't optional though as it's part of our release build.

Reply:
Could you instead apply a workaround for non-release installs?

Reply:
Sorry, I didn't mean for that second commit to be in this PR, only the mini issue with the catch() function.

Will propose a patch for the Makefile for non-release installs to build kovri-util instead.

I'm using Gentoo Linux amd64 and arm, want to package kovri.
Installing in Gentoo by default means to build from source, so the package-management is not much more then compile and install scripts in a chroot. Shell is bash.

Reply:
About kovri-util being optional or not:
In the Makefile, it's not used unless for release-builds.
The proposed change to the installer still installs kovri-util, it just does not check for its presence as required resource (which triggers the catch() function).

Should I create another PR where the Makefile builds kovri-util for non-release installs? Thus making kovri-util no longer 'optional'?